internal/v5/api.go

```diff
@@ -994,9 +994,11 @@ func checkExtraInfoKey(key string, field string) error {
 // GET id/meta/perm
 // https://github.com/juju/charmstore/blob/v4/docs/API.md#get-idmetaperm
 func (h *ReqHandler) metaPerm(entity *mongodoc.BaseEntity, id *router.ResolvedURL, path string, flags url.Values, req *http.Request) (interface{}, error) {
-	// TODO: choose appropriate ACLs based on currently appropriate channel
-	// for chosen entity or "channel" form value if specified.
```

Comment:
The "or "channel" form value if specified." part should still have its own TODO.